.env.jungle

```diff
@@ -45,18 +45,27 @@ HAPI_COINGECKO_API_TOKEN_ID=eos
 HAPI_REWARDS_TOKEN=EOS
 HAPI_SYNC_STATS_INTERVAL=60
 HAPI_SYNC_SCHEDULE_HISTORY_INTERVAL=86400
+HAPI_EOS_MECHANICS_INCLUDE_TRANSACTION=
+HAPI_EOS_STATE_HISTORY_PLUGIN_ENDPOINT=
+HAPI_EOS_GET_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT=
+HAPI_EOS_FAUCET_ACCOUNT_PASSWORD=
+GOOGLE_APPLICATION_CREDENTIALS=
+HAPI_RE_CAPTCHA_PROJECT_ID=
+HAPI_PUBLIC_RE_CAPTCHA_KEY=
+HAPI_CREATE_ACCOUNT_ACTION_NAME=
 
 #webapp
 PORT=3000
-REACT_APP_TAG=dev
+REACT_APP_VERSION=dev
 REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=EOS Jungle Testnet Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://ipfs.eosio.cr/ipfs/QmfQSA2Qnd6hF1dmnBcTaynp1723PAzY3CdMxeWz8yEdt3
```

Comment:
no longer using IPFS for this , use this URL instead

https://eosio.online/images/jungle.jpg